test/suites/test_crud.py

```diff
@@ -0,0 +1,751 @@
+            'success': {
+                'correct_input': {
+                    'args': ['tester', [['==', 'bucket_id', 100]], {'timeout': 10}],
+                },
+                'correct_output': {
+                    'scalar': 9,
+                },
+            },
+            'error': {
+                'exception_input': {
+                    'args': ['no-such-space-name', [['==', 'bucket_id', 100]], {'timeout': 10}],
+                },
+                'exception_output': {
+                    'str': [
+                            r'CountError: Space "no-such-space-name" doesn\'t exist', 
+                        ],
+                },
+            },
+        },
+        'crud_unflatten_rows': {
+            'success': {
+                'correct_input': {
+                    'args': [
+                        [
+                            [1, 100, 'Mike'], 
+                            [2, 100, 'Mike'], 
+                            [3, 100, 'Mike'], 
+                            [4, 100, 'Mike'], 
+                            [5, 200, 'Bill'], 
+                            [6, 300, 'Rob'],
+                        ],
+                        [
+                            {'name': 'id', 'type': 'unsigned'}, 
+                            {'name': 'bucket_id', 'type': 'unsigned'}, 
+                            {'name': 'name', 'type': 'string'}
+                        ],
+                    ],
+                },
+                'correct_output': {
+                    'scalar': [
+                        {'bucket_id': 100, 'name': 'Mike', 'id': 1}, 
+                        {'bucket_id': 100, 'name': 'Mike', 'id': 2}, 
+                        {'bucket_id': 100, 'name': 'Mike', 'id': 3}, 
+                        {'bucket_id': 100, 'name': 'Mike', 'id': 4}, 
+                        {'bucket_id': 200, 'name': 'Bill', 'id': 5}, 
+                        {'bucket_id': 300, 'name': 'Rob', 'id': 6},
+                    ],
+                },
+            },
+            'error': {
+                'exception_input': {
+                    'args': [[],[]],
+                },
+                'exception_output': {
+                    'str': [],
+                },
+            },
+        },
+        'crud_truncate': {
+            'success': {
+                'correct_input': {
+                    'args': ['tester'],
+                },
+                'correct_output': {
+                    'scalar': True,
+                },
+            },
+            'error': {
+                'exception_input': {
+                    'args': ['no-such-space-name', {'timeout': 10}],
+                },
+                'exception_output': {
+                    'str': [
+                            r'"no-such-space-name" doesn\'t exist', 
+                        ],
+                },
+            },
+        },
+        'crud_stats': {
+            'success': {
+                'correct_input': {
+                    'args': [],
```

Comment:
Let's add one more test with space name. Maybe it'll be convenient to extract specific handles from this test dict to some separate one (up to you).

Reply:
Done, now this test case passes the name of space to the testing function